labellab-flask/api/controllers/modelscontroller.py

```diff
@@ -0,0 +1,81 @@
+from flask.views import MethodView
+from flask import make_response, request, jsonify, current_app
+from flask_jwt_extended import jwt_required
+
+from api.models.Model import Model, optional_params
+from api.models.Projects import Project
+
+class Save(MethodView):
+    """
+    This method removes the access token on logout and stores the revoked token.
+    """
+
+    def post(self):
```

Comment:
Please add the `@jwt_required` here you may refer to the logout classes for the boilerplate :)